connectors/katalog/example/asset.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: katalog.m4d.ibm.com/v1alpha1
+kind: Asset
+metadata:
+  name: data-csv
+spec:
+  secretRef: 
+    name: data-csv-creds
+  details:
```

Comment:
Would endpoint be a better name than details?

Reply:
I actually created my own structure but then decided to have the initial PR be a copy of what we have today in the proto (so the word details is from there) and evolve it later though a design doc. The exception is the word security that was originally called dataset metadata, I changed to avoid confusion with the k8s resource metadata but I will switch back .